docs/config.rst

```diff
@@ -174,18 +197,21 @@ Here's an example from stacker_blueprints_, used to create a VPC::
         PrivateSubnets: 10.128.8.0/22,10.128.12.0/22,10.128.16.0/22,10.128.20.0/22
         CidrBlock: 10.128.0.0/16
 
-
 Parameters
 ==========
 
-Parameters are a Cloudformation concept that allow you to re-use an existing
-Cloudformation template, but modify its behavior by passing in different
+.. note::
+  Parameters have been deprecated in favor of Variables_ and will be
+  removed in a future release.
+
+Parameters are a CloudFormation concept that allow you to re-use an existing
+CloudFormation template, but modify its behavior by passing in different
 values.
 
 stacker tries to make working with Parameters a little easier in a few ways:
 
-YAML anchors & references
--------------------------
+Parameter YAML anchors & references
```

Comment:
These can still be used in blueprint variables, right?  It's more a yaml concept, than a stacker one.